integration_tests/CMakeLists.txt

```
@@ -1515,6 +1515,8 @@ RUN(NAME implicit_typing_04 LABELS gfortran llvmImplicit EXTRA_ARGS --implicit-i
 RUN(NAME implicit_typing_05 LABELS gfortran llvm  EXTRA_ARGS --implicit-typing)
 RUN(NAME implicit_typing_06 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc fortran EXTRA_ARGS --implicit-typing)
 
+RUN(NAME implicit_do_1 LABELS gfortran llvm c llvm_wasm llvm_wasm_emcc fortran EXTRA_ARGS --implicit-typing)
+RUN(NAME implicit_do_2 LABELS gfortran llvm c llvm_wasm llvm_wasm_emcc fortran EXTRA_ARGS --implicit-typing)
```

Since these are implicit typing tests, why not append them to the rest of the implicit typing tests?
```suggestion
RUN(NAME implicit_typing_07 LABELS gfortran llvm c llvm_wasm llvm_wasm_emcc fortran EXTRA_ARGS --implicit-typing)
RUN(NAME implicit_typing_08 LABELS gfortran llvm c llvm_wasm llvm_wasm_emcc fortran EXTRA_ARGS --implicit-typing)
```